### PR TITLE
Rec map format

### PR DESCRIPTION
**Pull request by lornaarmstrong**
- update formatting of the recommendation map to include an orange background box
- changed where the buttons are
- gave the page a title, in keeping with the other pages

Rendered UI:
Screenshot 2020-08-28 at 2 37 48 PM - Display 2

### CODE REVIEW

**Review by raniarusly — APPROVED** [review]
LGTM :D

**Comment by lornaarmstrong** [comment]
Here's the updated rendered UI:

Screenshot 2020-09-01 at 10 37 21 AM - Display 2

File: src/main/webapp/styles/recommendation-map.css [hunk 1]
```diff
@@ -61,21 +61,25 @@ h2, h3, p, label {
   text-align: center;
 }
```
Comment (MostafaGabriel) [inline src/main/webapp/styles/recommendation-map.css:62]: nit: I think we don't need this class anymore?

**Review by MostafaGabriel — APPROVED** [review]
LGTM :) Thanks Lorna!

File: src/main/webapp/styles/recommendation-map.css (hunk 1 above)
Comment (lornaarmstrong) [reply src/main/webapp/styles/recommendation-map.css:62]: Good spot, thanks!

**Pull request merged**